src/audio/crossover/crossover.c

```diff
@@ -618,7 +618,7 @@ static int crossover_process_audio_stream(struct processing_module *mod,
 					  struct output_stream_buffer *output_buffers,
```

Comment:
Add Fixes: 5bcbcf1 ("Audio: Crossover: Convert to module adapter") to commit message.

Reply:
Still not good,

/zep_workspace/sof/src/audio/crossover/crossover.c:656:58: warning: incorrect type in argument 2 (different address spaces)

Seems avoiding sparse errors needs a different approach